### PR TITLE
nexttrace-bin: Add auto setcap install script

### PR DESCRIPTION
**Pull request by chise0713**

### CODE REVIEW

**Comment by ouuan** [comment]
Thanks for your contribution. You can add new files instead of using `cat`.

File: nexttrace-bin/PKGBUILD [hunk 1]
```diff
@@ -14,5 +14,6 @@ source=("$pkgname-$pkgver::$url/releases/download/$_tag/nexttrace_linux_amd64")
 sha256sums=('c8159dd7e7444bf937665f8c243c938a782a31e6102cc715179bca84f2ad7d0c')
 
 package() {
+    install -Dm644 ./nexttrace.hook "$pkgdir/usr/share/libalpm/hooks/nexttrace.hook"
```
Comment (ouuan) [inline nexttrace-bin/PKGBUILD]:
```suggestion
    install -Dm644 "$srcdir/nexttrace.hook" "$pkgdir/usr/share/libalpm/hooks/nexttrace.hook"
```

**Comment by ouuan** [comment]
I just realized that we should check for the best practice. [Wireshark](https://gitlab.archlinux.org/archlinux/packaging/packages/wireshark) and [mtr](https://gitlab.archlinux.org/archlinux/packaging/packages/mtr) use install scripts instead of pacman hooks. What do you think? Do you have any other references?

**Comment by chise0713** [comment]
I think yes, we should use install scripts, instead pacman hooks, my mistake.
I think I don't have other references use install scripts, I just learned there is such a thing.

**Pull request merged**